portal-api/src/v1/users/routes.js

```diff
@@ -193,6 +193,66 @@ module.exports.resetPassword = async (req, res) => {
 
 module.exports.resetPasswordWithToken = async (req, res, next) => {
   const { resetPwdToken } = req.params;
+  const { currentPassword, password, passwordConfirm} = req.body;
+
+  if (currentPassword.trim() === '') {
+    return res.status(200).json({
+      success: false,
+      errors: {
+        count: 1,
+        errorList: {
+          currentPassword: {
+            text: 'Empty password',
+          },
+        },
+      },
+    });
+  }
+
+  if (password.trim() === '') {
+    return res.status(200).json({
+      success: false,
+      errors: {
+        count: 1,
+        errorList: {
+          password: {
+            text: 'Empty password',
+          },
+        },
+      },
+    });
+  }
+
+  if (passwordConfirm.trim() === '') {
+    return res.status(200).json({
+      success: false,
+      errors: {
+        count: 1,
+        errorList: {
+          passwordConfirm: {
+            text: 'Empty password',
+          },
+        },
+      },
+    });
+  }
+
+  if (password.trim() !== passwordConfirm.trim()) {
+    return res.status(200).json({
+      success: false,
+      errors: {
```

Comment:
Should we be returning 400 status code for all of these added responses instead of 200? Because errors are returned.